### PR TITLE
New engineering page

### PR DESCRIPTION
**Pull request by lucasrenan**
- creates a fromAtoB theme
- move the old html and static code to `old` folders. it may be deleted in a next PR.

home:

screen shot 2017-04-11 at 19 52 08

post:

screen shot 2017-04-11 at 19 52 22

### CODE REVIEW

**Comment by 23inhouse** [comment]
I think the `meetup` and `blog` should be a link at the top next to `engineering`

**Comment by actfong** [comment]
Looks great! 👍

**Comment by 23inhouse** [comment]
@herrtreas you might like to check this.

**Comment by 23inhouse** [comment]
How can we see all the blog posts?

File: themes/fromAtoB/layouts/index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    {{ partial "nav.html" . }}
+
+    <header class="home-header">
+      <div class="header-content">
+        <div class="header-content-inner">
+          <h1>Help people find the best way to go!</h1>
+          <hr>
+          <p>
+            You love to travel, are full of ambition, and want to make a big impact? At fromAtoB we're helping people find the fastest and cheapest way to get from A to B.
```
Comment (23inhouse) [inline themes/fromAtoB/layouts/index.html]: Should we be focusing on traveling or working in an awesome team that gets to work in a self guided way?

File: themes/fromAtoB/layouts/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            <a href="https://www.meetup.com/Vue-js-Berlin/" rel="nofollow" class="btn btn-primary btn-xl sr-button" target="_blank">Check out Meetup Page</a>
+          </div>
+
+          <div class="col-lg-5 text-center">
+            <h2 class="section-heading">Blog</h2>
+            <hr class="primary">
+            <p>
+              We maintain an engineering blog where we post some cool stuff.
+            </p>
+            {{ range first 1 (where .Data.Pages "Type" "post") }}
```
Comment (mauricioklein) [inline themes/fromAtoB/layouts/index.html]: Is this correct, pointing to the first post on blog?
Don't we have a posts home page, like `/posts`?

File: themes/fromAtoB/layouts/index.html (hunk 1 above)
Comment (herrtreas) [inline themes/fromAtoB/layouts/index.html]: I think we can keep this here for the time being and iterate when its out

**Comment by lucasrenan** [comment]
@23inhouse we may update the texts, we didn't have anything better for the first version.

@mauricioklein I did this little hack, but you are totally right, we must implement a `/posts` page.

**Pull request merged**